index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="G:\GIT WORKSPACE\HELLO_WORLD_PROJECT\style.css" >
+        <title>Welcome Microverse </title>
+    </head>
+        <body>
+            <h1 class="color1">Hello Microverse!</h1>
```

Comment:
[Optional] This is a class. In the css you used `#color1` which is an id selector instead of `.color1` the class selector.